packages/sdks/src/blocks/accordion/accordion.lite.tsx

```diff
@@ -177,23 +177,25 @@ export default function Accordion(props: AccordionProps) {
               class={state.getAccordionTitleClassName(index)}
               style={{
                 ...state.accordionTitleStyles,
-                width: props.grid ? props.gridRowWidth : undefined,
-                ...(useTarget({
-                  reactNative: {},
-                  default: {
-                    order:
-                      state.openGridItemOrder !== null
-                        ? convertOrderNumberToString(index)
-                        : convertOrderNumberToString(index + 1),
-                  },
-                }) as any),
+                ...(props.grid && {
+                  width: props.gridRowWidth,
+                  ...(useTarget({
+                    reactNative: {},
+                    default: {
+                      order:
+                        state.openGridItemOrder !== null
+                          ? convertOrderNumberToString(index)
+                          : convertOrderNumberToString(index + 1),
+                    },
+                  }) as any),
+                }),
               }}
               data-index={index}
               onClick={() => state.onClick(index)}
             >
               <Blocks
                 blocks={item.title}
-                path={`items.${index}.title`}
+                path={`component.options.items.${index}.title`}
```

Comment:
actually can we improve `Blocks` to also add the prefix if its not there? then we can simplify code for end users building with `Blocks`

Reply:
yup sure!